test/hotspot/jtreg/compiler/loopopts/superword/MinMaxRed_Long.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, Red Hat, Inc. All rights reserved.
```

Comment:
Should probably be 2025 by now, right? At least that is our policy for Oracle, not sure what you want to do at Red Hat